### DISCUSSION TITLE
Could we not setup a server Discussion on Discord

### DISCUSSION THREAD

**Discussion by Atlas-Night-Out** [NightscoutFoundation/xDrip#2620] [Q&A]
Would or could it be possible for us all to have it located on Discord for our discussions and issues?
I find it very handy for AAPS things and Nightscout but do not see one for xdrip+

Whats everyone's else's thoughts on this?

**Comment by Navid200** [comment]
What is the advantage of that over what we have here in these discussions?

**Reply by DiabeticDan** [reply]
Discord would have instant messaging, replying, easy access to person-2-person live chat. You can also share screen on your device, or just use a microphone on your computer or phone to chat, so you can walk someone through a setup/troubleshooting process much quicker, and in some cases more precise than text on a wall.

**Reply by DiabeticDan** [reply]
It also has a robust search as well, so that wont be missed out on. Also you can webhook the Github to the Discord, so updates and discussions wont be missed either.

https://ardalis.com/integrate-github-and-discord-with-webhooks/

**Reply by Navid200** [reply]
We opened these discussions because people used to open issues just to ask questions.  
That was not good because it would take a very long time to constantly have to go through all the invalid issues and label them as such.

Since we have opened these discussions, we are also using them to make announcements.
So, it is really useful for communicating with everyone.  A user can post here to reach developers.  A developer can post here to reach the user community.
A user can post here to reach other users.

What is missing is not that much.  I wish it offered private rooms.  It doesn't.

But, we have never needed a platform that would allow us to have live chats.  I mean there may be many features offered by Discord that's missing here.   But, if we don't need it, it makes no difference.
I don't see the value in a conversation between only two people that is going to disappear as soon as the conversation is over.   The value of a forum is that conversations remain on the record and can be helpful to others who may come in the future and see.

I hear occasionally from people saying that developers are on Discord.  I ask why?  And I am yet to hear what's wrong with GitHub.  I have nothing against Discord.   But, what's the point of moving if what we have does what we need?

**Reply by 4RK4N** [reply]
im just gonna throw "users helping users" into the benefits. like at least 50% of your discussion tickets would solve itself by that, leaving you more time for the "real" problems.

honestly if you really question the "use" of a live chat compared to a static forum then its maybe time to arrive in the 21st century finally.
but yes discord doesnt send you emails on every message and requires a little less "i answer that somewhen next week/month/year or never" attitude.

i mean xdrip is good no prob there but being a little closer to your users wont hurt at all. and a discord is the method of choice for that.

**Reply by Navid200** [reply]
Many users are helped by discussions that are on public forums and you may never hear from those.  Just because you don't hear from them does not mean that it doesn't exist.  All of these discussions are indexed by Google.
Are Discord conversations indexed by Google?

Those who are volunteers here don't have much time.  I want their time to be used to the maximum.  I don't believe any of them has time to spend on a one-to-one basis.  
If a developer takes the time to explain something, I would like their explanation to be on the record for everyone else who may benefit.

I have tried to explain what the benefits of these discussion are.  But, in return I have got sarcasm.  I am not going to respond if that continues.

**Reply by 4RK4N** [reply]
as if milos or any of the aaps devs spend 24/7 time online in dc and as if he or any of the devs is walking newcomers through the basic setups in 1-on-1s ... see how it works there.
like i said at least 50% of your support discussions would solve itself without any interaction on your side (and not remain unsolved for months/years). only the big unsolvable probs would then remain for the dev team.

besides making an account on a "developer" website (yes ppl see github as just that) is still a high bar for many people like "its a dev site, im no dev, i have nothing to say there" while discord is just like facebook everyone can do it and there is no mental high bar to get over.

and since you are part of nightscout foundation somehow you sure can get a channel or a group of channels in the aaps discord as well. there are more than enough people there using xdrip who can help with the most trivial problems leaving more dev time for the important things.

**Reply by Navid200** [reply]
If xDrip users want to help each other, they can.  And they can choose to use whatever tools they like for that.  If they want to use Discord, they can.
I chose to use xDrip users on facebook about 5 years ago and still am present there.
I was invited to join GitHub because of that.  So, I do what I can here to help as well.

I have created an xDrip server on Discord and offered it to the developers as an option.
There has never been an interested.

Over the years, I have learnt that you cannot ask a volunteer to do something.  You can suggest.  If they agree, they will do it.  if you charge it with a statement like "... the 21st century", it is very likely they will not even bother looking at the conversation a second time.

**Reply by arjungm** [reply]
@Navid200 do you still have the xdrip users discord?

**Reply by Navid200** [reply]
I created a server.  But, it has never been used.

**Reply by Der-Schubi** [reply]
> I created a server. But, it has never been used.

Then why not make a xDrip channel on the #WeAreNotWaiting Discord (https://discord.gg/gZ5ze5wX)?
There are questions about xDrip regularly, being able to ping you would be helping to raise some issues or clarify things.
And as 4RK4N said, many problems would be solved without a dev, just by the users there.

**Reply by Navid200** [reply]
Is that something I can do?  Or, do I need someone's permission?

**Reply by Der-Schubi** [reply]
> Is that something I can do? Or, do I need someone's permission?

I have contact to one of the servers admins (cluckj). If you want, I can ask him.

**Reply by Navid200** [reply]
No, there is no need.
I don't know why people cannot ask their questions here.

**Reply by Der-Schubi** [reply]
Of course, they could. There may be a bit of hesitation to write here, as 4RK4N said.
The really positive thing about such a Discord channel would be the synergy effect, having all the AAPS users already on this server. Most of them are using xDrip.

**Reply by Navid200** [reply]
What is significantly important about here is that the xDrip lead developer can see the conversations.  
Opening another platform that he is not going to be a part of is not really going to help anyone.

**Reply by jvenge** [reply]
Discord is fleeting messages that get lost in oblivion. Plus data privacy issues. Prefer to keep it here, as limited as it is, but email notifications make sure I don't miss things.
